src/main/java/dev/codesupport/web/api/data/repository/ShowcaseRepository.java

```diff
@@ -0,0 +1,12 @@
+package dev.codesupport.web.api.data.repository;
+
+import dev.codesupport.web.api.data.entity.ShowcaseEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ShowcaseRepository extends JpaRepository<ShowcaseEntity, Long> {
+
+
+
+}
```

Comment:
Empty body?

Reply:
Repos come stocked with default methods, just need to declare it to begin with to have access to using it.